src/ingestion/sources/base.py

```diff
@@ -314,7 +314,7 @@ def _transform_col_info_metric(self, col_name, metric, table_name):
             raise Exception("Unreachable: Metric type is defined that does not resolve to a definition.")
 
         select_unformatted = attr()
-        select_no_alias = select_unformatted.format('"{}"'.format(col_name.upper()))
+        select_no_alias = select_unformatted.format('{}'.format(col_name))
```

Comment:
The reason for this change is because column names that were lowercase were not being retrieved. Hence, IMO It is more extensible to keep this lowercase in base. Snowflake works with this approach.

Reply:
Tested it with Snowflake connections as well which is why I mentioned that it works. Ultimately Snowflake allows you to write a query with table name in lowercase and it automatically converts it to uppercase. Here's a thread from dbt on the topic - https://discourse.getdbt.com/t/table-field-syntax-best-practices-when-using-snowflake/33/2